foreman-installer/foreman-installer.spec

```diff
@@ -16,15 +16,15 @@ Source0:    http://downloads.theforeman.org/%{name}/%{name}-%{version}%{?dashalp
 BuildArch:  noarch
 
 Requires:   curl
-Requires:   %{?scl_prefix}puppet-agent >= 1.6.1
+Requires:   %{?scl_prefix}puppet-agent >= 1.9.0
```

Comment:
Would puppet-agent be in the SCL? I know we build this without SCL so it's probably fine for now.

Reply:
no idea.

Reply:
It's not in the SCL and with the way its built I wouldn't ever forsee it being in so we could drop that.